config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    emberResponsiveDefaults: {
```

Comment:
Ya, I had `responsiveDefaults` but I felt that was iffy because it almost sounds like it could be `breakpoints`. `emberResponsive` is probably best.